locations/pipelines/drop_attributes.py

```diff
@@ -0,0 +1,17 @@
+from scrapy import Spider
+
+from locations.items import Feature
+
+
+class DropAttributesPipeline:
+    def process_item(self, item: Feature, spider: Spider):
+        if not hasattr(spider, "drop_attributes"):
+            return item
+
+        for attribute in getattr(spider, "drop_attributes"):
+            if attribute in item.fields:
+                del item[attribute]
```

Comment:
I think del throws an exception thi the attribute isn't present. Add an if or switch to None

Reply:
Use `item.pop(attribute, None)` instead?